src/releasers/node.ts

```diff
@@ -154,4 +154,8 @@ export class Node extends ReleasePR {
     }
     return this.pkgJsonContents;
   }
+
+  protected defaultInitialVersion(): string {
+    return this.bumpMinorPreMajor ? '0.1.0' : super.defaultInitialVersion();
```

Comment:
@yi-Xu-0100 I think we can move this into the super class, this behavior should be supported by all languages.